### ISSUE TITLE
Fine-tune the learning rate scheduler

### ISSUE THREAD

**Issue by vc1492a** [vc1492a/tidd#77]
We use a learning rate to help minimize over fitting but the parameters need to be tuned after balancing the data and selecting an architecture.
